src/dex/camelot/config.ts

```diff
@@ -19,7 +19,6 @@ export const CamelotConfig: DexConfigMap<DexParams> = {
 
 export const Adapters: Record<number, AdapterMappings> = {
   [Network.ARBITRUM]: {
-    [SwapSide.SELL]: [{ name: 'ArbitrumAdapter02', index: 1 }],
-    [SwapSide.BUY]: [{ name: 'ArbitrumBuyAdapter', index: 1 }],
+    [SwapSide.SELL]: [{ name: 'ArbitrumAdapter02', index: 5 }],
```

Comment:
We were using the same Adapter as for Solidly but as it calls `pool.stable()`  which fails because the function name in Camelot pair is` stableSwap()`

Reply:
new adapter: https://arbiscan.io/address/0xD1F70c98a78d48A93F0B4dDa49057469dc5aC126#code